doc/modules/linear_model.rst

```diff
@@ -838,7 +838,7 @@ Perceptron
 ==========
 
 The :class:`Perceptron` is another simple algorithm suitable for large scale
-learning. By default:
+learning. It is a binary classifier. By default:
```

Comment:
I would just say: 

> The Perceptron is another simple classification algorithm suitable for large scale learning. By default:

Reply:
Thanks for the suggestion. I have done the necessary changes